src/avram/queryable.cr

```diff
@@ -243,6 +243,22 @@ module Avram::Queryable(T)
     end
   end
 
+  # Remove when PG::PGValue matches rs.read possibilities
+  alias PGValue = Bool | Float32 | Float64 | Int32 | Int64 | PG::Numeric | String | Time | UUID | Nil
```

Comment:
👍  good call. Can we add `Int16` too? That's the only other simple type that Avram supports missing from here.

Reply:
```suggestion
  alias PGValue = Bool | Float32 | Float64 | Int16 | Int32 | Int64 | PG::Numeric | String | Time | UUID | Nil
```